java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -81,7 +81,7 @@ abstract class AbstractArcHost(
     override val hostId = "${this::class.className()}@${this.hashCode()}"
 
     // TODO: refactor to allow clients to supply this
-    private val coroutineContext = Dispatchers.Unconfined + CoroutineName("AbstractArcHost")
+    open val coroutineContext = Dispatchers.Unconfined + CoroutineName("AbstractArcHost")
```

Comment:
We could probably move this to a default constructor param, but for now I guess this is ok.